app/components/LiveDock.vue.ts

```diff
@@ -297,7 +290,9 @@ export default class LiveDock extends Vue {
     const minEditorWidth = 860;
     const minWidth = 290;
     const maxWidth = Math.min(appRect.width - minEditorWidth, appRect.width / 2);
+    // tslint:disable-next-line:no-parameter-reassignment TODO
     width = Math.max(minWidth, width);
+    // tslint:disable-next-line:no-parameter-reassignment
```

Comment:
this also seems indicative of an actual issue that should be handled rather than ignoring rules here

Reply:
This code is pretty old, but I think it should be safe to just use a new local variable here.

Reply:
About parameter reassignment, there were too many instances of this for me to feel comfortable rewriting at this moment, I've added TODO comments but can look into it in a smaller PR (like the commits I've split from it already).

Reply:
k